### PR TITLE
completed task 3

### PR DESCRIPTION
**Pull request by sophynez**
Task 3 consists on  implementing a simple MLP with batch normalization and dropout for modeling uncertainty.
branch consists of : 
#### in src/models 
- MLPModel.py : definition of MLP model in pytorch
- mlp_train.py : train MLP on the international airlines passengers dataset
- mlp_uncertainty.py : inference and uncertainty visualization
#### in Docs
- report summarizing he work done

### CODE REVIEW

**Comment by 1hachem** [comment]
great job 🚀 

1- try to drop some data in the middle to visualize how the model behaves in unseen regions (epistimic uncertainty)
2- change your saved models paths to `model/`, and use an UPPER_CASE naming of the paths since its a constant, put it on top
3- apply what you can learn from this https://goodresearch.dev/
4- make sure you learn `./scripts/setup.py` before anything as it will setup `pre-commit` hooks and format your code

**Comment by 1hachem** [comment]
going to close this since it has been 2 weeks with no activity
